.github/workflows/client.yml

```diff
@@ -57,4 +57,27 @@ jobs:
       - run: ./gradlew test
         working-directory: ./client/ionic/android
       - run: ./gradlew build
-        working-directory: ./client/ionic/android
\ No newline at end of file
+        working-directory: ./client/ionic/android
+  # Builds the iOS app
+  build-ios-debug:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+      - run: npm install -g @ionic/cli
+      - run: npm install
+        working-directory: ./client/ionic
+      - run: ionic build
+        working-directory: ./client/ionic
+      - run: ionic capacitor copy ios
```

Comment:
Why do we run `ionic cap sync` for iOS but not Android -- and `ionic capacitor copy` for Android but not iOS?

Reply:
I'm gonna merge this for now, if it works it works. 
We will later revisit & optimize CI anyway since building for every commit in every PR is too slow.